.github/workflows/ci-kickoff.yml

```diff
@@ -0,0 +1,54 @@
+name: CI Kickoff
+
+on:
+  merge_group:
+  push:
+    branches:
+      - master
+      - 'dev-gh-*'
+  pull_request:
+    branches-ignore:
+      - hotfix-* # This is to ensure that this workflow is not triggered twice on ic-private, as it's already triggered from release-testing
+
+jobs:
+  ci-main:
+    name: CI Main
+    # run on all events except for forked PRs
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+    uses: ./.github/workflows/ci-main.yml
+    secrets: inherit
+    with:
+      commit-sha: ${{ github.sha }}
+    permissions:
+      contents: read
+      pull-requests: read
+
+  ci-pr-only:
+    name: CI PR Only
+    # only run on PRs and not forked PRs
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
[The original ci-pr-only workflow](https://github.com/dfinity/ic/blob/b3892b04754288f08e20e2b41fe9ebde1d3d667a/.github/workflows/ci-pr-only.yml#L7) was only triggered on `opened`, `synchronize` or `reopened` PR event types. Does it make sense to further constrain this condition to only call `ci-pr-only.yml` when `github.event.type` is one of those types?

Otherwise we would also be running that workflow if the PR title or description is edited for example, or when labels are added.